qa/restart-upgrade/src/test/java/org/opensearch/neuralsearch/bwc/SemanticSearch.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.neuralsearch.bwc;
+
+import com.carrotsearch.randomizedtesting.RandomizedTest;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.opensearch.neuralsearch.TestUtils.NODES_BWC_CLUSTER;
+
+import static org.opensearch.neuralsearch.TestUtils.getModelId;
+import static org.opensearch.neuralsearch.TestUtils.TEXT_EMBEDDING_PROCESSOR;
+import org.opensearch.neuralsearch.query.NeuralQueryBuilder;
+
+public class SemanticSearch extends AbstractRestartUpgradeRestTestCase {
```

Comment:
Can this be `SemanticSearchIT` to be consistent with out test names?

Reply:
ACK